README.md

```diff
@@ -6,12 +6,12 @@
 
 ## Setup
 
-- Visit painite-tc repo at ```https://github.com/technical-council-nitt/painite-tc```
```

Comment:
The 3 backticks denote code in the markdown file. That's why I added the 3 backticks for links, websites or commands.
Just now, discovered both 3 backticks and single backticks work same. :+1: :100:

Reply:
Yeah it just auto edits